web-api/src/persistence/elasticsearch/getCaseInventoryReport.ts

```diff
@@ -1,13 +1,13 @@
+import { CASE_INVENTORY_PRINT_REPORT_MAX_SIZE } from '@shared/business/entities/EntityConstants';
 import { QueryDslQueryContainer } from '@opensearch-project/opensearch/api/types';
 import { search } from './searchClient';
-
 /**
  * getCaseInventoryReport
  *
  * @param {object} providers the providers object
  * @param {object} providers.applicationContext the application context
  * @param {string} providers.associatedJudge the optional judge filter
- * @param {number} providers.from the item index to start from
+ * * @param {number} providers.from the item index to start from
```

Comment:
This looks like it was probably unintentional?

Even so, I think this entire jsdoc block can be removed since this method is typed.